server/game/cards/events/06/savingthekingdom.js

```diff
@@ -0,0 +1,40 @@
+const DrawCard = require('../../../drawcard.js');
+
+class SavingTheKingdom extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Make character unable to stand',
+            phase: 'challenge',
+            target: {
+                activePromptTitle: 'Select a character',
+                cardCondition: card => card.location === 'play area' && card.getType() === 'character' && card.getStrength() <= 3
+            },
+            handler: context => {
+                this.untilEndOfRound(ability => ({
```

Comment:
Oh, interesting. In that case, yes, you were correct to leave off the `gameAction`, as that will make you unable to target a character that couldn't be kneeled.